pkg/pipelines/rules/policies/untrusted_input_usage.rego

```diff
@@ -25,7 +25,7 @@ deny[result] {
 	count(path) == 1
 	pipeline.contains_untrusted_inputs(value)
 	result := {
-		"msg": value,
+		"msg": sprintf("consider evaluating user inputs in intermediate environment variables instead of using them right in '%s' step shell", [input.jobs[i].steps[j]]),
```

Comment:
This one starts with a lower case letter, the one above starts with a capital letter.